tests/unit/db/db_updates_tests.py

```diff
@@ -0,0 +1,469 @@
+        self.client.disconnect()
+        super(DbUpdatesTestsBase, self).tearDown()
+
+    def create_dbs(self, count=3):
+        self.new_dbs += [(self.client.create_database(self.dbname())) for x in range(count)]
+
+@unittest.skipIf(os.environ.get('RUN_CLOUDANT_TESTS'),
+    'Skipping CouchDB _db_updates feed tests')
+class CouchDbUpdatesTests(DbUpdatesTestsBase):
+    """
+    CouchDB _db_updates feed unit tests
+    """
+    def test_constructor_db_updates(self):
+        """
+        Test constructing a _db_updates feed
+        """
+        feed = Feed(self.client, feed='continuous', heartbeat=False, timeout=2)
+        self.assertEqual(feed._url,
+            '/'.join([self.client.cloudant_url, '_db_updates']))
+        self.assertIsInstance(feed._r_session, Session)
+        self.assertFalse(feed._raw_data)
+        self.assertDictEqual(feed._options,
+            {'feed': 'continuous', 'heartbeat': False, 'timeout': 2})
+
+    def test_stop_iteration_of_continuous_feed_with_heartbeat(self):
+        """
+        - Test stopping the iteration
+        - Test continuous feed
+        - Test heartbeat
+        """
+        feed = Feed(self.client, feed='continuous', timeout=100)
+        changes = list()
+        for change in feed:
+            if not change:
+                if not self.new_dbs:
+                    self.create_dbs(5)
+                else:
+                    continue
+            else:
+                changes.append(change)
+                if len(changes) == 3:
+                    feed.stop()
+        self.assertEqual(len(self.new_dbs), 5)
+        self.assertEqual(len(changes), 3)
+        self.assertDictEqual(
+            changes[0], {'db_name': self.new_dbs[0].database_name, 'type': 'created'})
+        self.assertDictEqual(
+            changes[1], {'db_name': self.new_dbs[1].database_name, 'type': 'created'})
+        self.assertDictEqual(
+            changes[2], {'db_name': self.new_dbs[2].database_name, 'type': 'created'})
+
+    def test_get_raw_content(self):
+        """
+        Test getting raw feed content
+        """
+        feed = Feed(self.client, raw_data='True', feed='continuous', timeout=100)
+        raw_content = list()
+        for raw_line in feed:
+            self.assertIsInstance(raw_line, BYTETYPE)
+            if not raw_line:
+                self.create_dbs(3)
+            else:
+                raw_content.append(raw_line)
+                if len(raw_content) == 3:
+                    feed.stop()
+        changes = [json.loads(unicode_(x)) for x in raw_content]
+        self.assertDictEqual(
+            changes[0], {'db_name': self.new_dbs[0].database_name, 'type': 'created'})
+        self.assertDictEqual(
+            changes[1], {'db_name': self.new_dbs[1].database_name, 'type': 'created'})
+        self.assertDictEqual(
+            changes[2], {'db_name': self.new_dbs[2].database_name, 'type': 'created'})
+
+    def test_get_longpoll_feed_as_default(self):
```

Comment:
In CouchDB `_db_updates` works a bit differently from `Cloudant`.  There is no "since" option and a `longpoll` feed always starts from the point from which the request is made to the server.  As such, there is no point to prime the server with any new databases because they would be irrelevant to the `longpoll` request.  Also, the `longpoll` behavior appears to be a "one and done" type situation.  Meaning that a `longpoll` feed is waiting for a single "event" to occur after which it terminates and a heartbeat constitutes an event, so there is no opportunity to create a database as part of the feed loop either since the first event will always be a heartbeat in this test.